gc/base/MarkingScheme.cpp

```diff
@@ -272,14 +294,15 @@ MM_MarkingScheme::setMarkBitsInRange(MM_EnvironmentBase *env, void *heapBase, vo
 void
 MM_MarkingScheme::completeScan(MM_EnvironmentBase *env)
 {
-	omrobjectptr_t objectPtr = NULL;
-	MM_WorkPackets *packets = getWorkPackets();
-
+	uintptr_t *bytesScanned = &(env->_markStats._bytesScanned);
```

Comment:
Why are the stats just not being updated in the scanObject call?  Also why are they being cached locally? I feel like that confuses the code.  The compiler should be generating the same code even if you find them via the env everytime.